resources/deployments/mongo/deployments.go

```diff
@@ -261,6 +261,9 @@ func buildStatusQuery(status deployments.StatusQuery) bson.M {
 					bson.M{
 						buildStatusKey(deployments.DeviceDeploymentStatusRebooting): gt0,
 					},
+					bson.M{
+						buildStatusKey(deployments.DeviceDeploymentStatusAlreadyInst): gt0,
```

Comment:
not sure if this will do the trick, consider object  with:

```
already-installed:1
everything else: 0
```

This would end up in results of `inprogress`  while should be in finished.

```
pending: 1
success: 1
```

This should end up in `inprogress` results while it won't be included.